src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends React.PureComponent {
+  render() {
+    const { name } = this.props;
+    return (
+      <button type="button">{ name }</button>
+    );
+  }
+}
```

Comment:
[Optional]

This can be a Function Component since it's stateless. 
I think something like this should be enough. :thinking: 

```jsx
const Button = ({ name }) => (<button type="button">{ name }</button>);
```

The `name` prop has been destructured from the param.

Reply:
Okay, I'll change it :)